### ISSUE TITLE
"Retrying setup: Cannot reopen a client instance, once it has been closed."

### ISSUE THREAD

**Issue by mvinci-wyze** [natekspencer/hacs-litterrobot#13]
### The problem
The title of this post is the error that shows up on the Integration Tile.

actual home-assistant.log lines:

`WARNING (MainThread) [homeassistant.config_entries] config entry '<EMAIL>' for literrobot integration not ready yet: Cannot reopen a client instance, noce it has been closed.; Retrying in background`

### What is version of Home Assistant Core has the issue?
2021.9.0b5

### What was the last working version of Home Assistant Core?
n/a

### What type of installation are you running?
Home Assistant OS

### Integration causing the issue
Litter-robot

### Link to integration documentation on our website
https://www.home-assistant.io/integrations/youless/

### Anything in the logs that might be useful for us?

edt: post copied from ha issue post

**Comment by f3bandit** [comment]
what was the fix if any for this issue? as I have the same issue?

**Comment by f3bandit** [comment]
2021-09-04 19:36:40 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration alexa_media which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-04 19:36:40 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-04 19:36:40 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration litterrobot which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-04 19:36:46 ERROR (MainThread) [snitun.client.client_peer] Challenge/Response error with SniTun server
2021-09-04 19:36:46 ERROR (MainThread) [hass_nabucasa.remote] Connection problem to snitun server
2021-09-04 19:37:22 WARNING (MainThread) [homeassistant.config_entries] Config entry 'f3bandit@gmail.com' for litterrobot integration not ready yet: Cannot reopen a client instance, once it has been closed.; Retrying in background
2021-09-04 19:38:27 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: hacs
2021-09-04 19:39:36 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: life360, alexa_media, device_tracker
2021-09-04 19:46:58 ERROR (MainThread) [custom_components.litterrobot.hub] Invalid credentials
2021-09-04 19:47:30 WARNING (MainThread) [homeassistant.config_entries] Config entry 'f3bandit@gmail.com' for litterrobot integration not ready yet: Cannot reopen a client instance, once it has been closed.; Retrying in background
